config/src/main/main.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
should .gitignore these probably and remove from the PR

Reply:
🤦  Will remove this.